src/App.js

```diff
@@ -14,11 +14,70 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [...goodsFromServer],
+    selectGoods: [],
+  };
+
+  changeGoods = (good, sel) => {
```

Comment:
```suggestion
  changeGoods = (good, selected) => {
```
give more meaningful names